src-tauri/src/lib.rs

```diff
@@ -234,10 +236,16 @@ pub fn run() {
             uninstall_game,
             // Processes
             launch_game,
-            kill_game
+            kill_game,
+            toggle_autostart,
+            get_autostart_enabled,
         ])
         .plugin(tauri_plugin_shell::init())
         .plugin(tauri_plugin_dialog::init())
+        .plugin(tauri_plugin_autostart::init(
+            tauri_plugin_autostart::MacosLauncher::LaunchAgent,
```

Comment:
MacOS launcher?

Reply:
@DecDuck 
![image](https://github.com/user-attachments/assets/0e389929-5472-4e39-bdfc-34e8098e6c2e)
future proofing, its just the name they used in the documentation

https://v2.tauri.app/plugin/autostart/

Reply:
i can remove it i guess?

Reply:
Oh so it's not a MacOS specific thing, good.